### PR TITLE
compiler configuration from URL e2e tests

### LINKED ISSUE #533: Add e2e tests to check compiler configuration through URL
Test `optimize`, `evmVersion` & `version` setting and updating using URL

Test `version` for custom compiler URL too.

Related: #532

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #533

### CODE REVIEW

**Review by cleytontiagosorinho — COMMENTED** [review]
Muito bom

**Review by cleytontiagosorinho — CHANGES_REQUESTED** [review]
Sim por favor

File: apps/remix-ide-e2e/src/tests/url.test.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    .url('http://127.0.0.1:8080/#version=https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js')
+    .refresh()
+    .pause(5000)
+    .clickLaunchIcon('solidity')
+    .pause(5000)
+    .assert.containsText('#versionSelector option[selected="selected"]', 'custom')
+    // default values
+    .verify.elementPresent('#optimize')
+    .assert.elementNotPresent('#optimize:checked')
+    .assert.containsText('#evmVersionSelector option[selected="selected"]', 'default')
```
Comment (yann300) [inline apps/remix-ide-e2e/src/tests/url.test.ts:39]: the test suite should end with `.end()`

**Review by yann300 — APPROVED** [review]

**Review by cleytontiagosorinho — APPROVED** [review]
4843bd3e0987e68b27553422f001a6b406b77ec7

**Pull request merged**